src/lib/lightning/eclair/eclairService.spec.ts

```diff
@@ -73,6 +73,7 @@ describe('EclairService', () => {
       state: ELN.ChannelState.NORMAL,
       data: {
         commitments: {
+          channelFlags: 1,
```

Comment:
Added this because I updated the type from `any` to a more specific one so I had to add the missing property to the test. Channel flag `1` in `eclair` is for public channels.

Reply:
Great improvement. I should have done this initially instead of using `any`. 🥇